3-extra/1-factorial.js

```diff
@@ -8,20 +8,29 @@
     Using a loop, complete the function below so it returns the factorial of the number being passed in. 
 */
 
-function factorial(input) {
-    // TODO
-}
+const factorial = (input) => {
```

Comment:
Great job. As a side bar, factorials are very useful for trying to count how many different orders there are for things  or how many different ways we can combine things.